src/openforms/prefill/contrib/objects_api/plugin.py

```diff
@@ -37,3 +40,11 @@ def get_config_actions(self):
                 ),
             ),
         ]
+
+    @property
+    def extra_data(self):
+        return {
+            "api_groups": [
+                (group.pk, group.name) for group in ObjectsAPIGroupConfig.objects.all()
```

Comment:
```suggestion
                (group.pk, group.name) for group in ObjectsAPIGroupConfig.objects.iterator()
```
to avoid loading the entire result set in memory, minor optimization but it's for free here